src/vcpkg/sourceparagraph.cpp

```diff
@@ -78,20 +78,20 @@ namespace
                                  "parenthesized expressions).");
     DECLARE_AND_REGISTER_MESSAGE(LicenseExpressionExpectCompoundOrWithFoundWord,
                                  (msg::value),
-                                 "",
+                                 "Example of {value} is 'MIT'",
                                  "Expected either AND, OR, or WITH, found a license or exception name: '{value}'.");
     DECLARE_AND_REGISTER_MESSAGE(LicenseExpressionExpectCompoundFoundWord,
                                  (msg::value),
-                                 "",
+                                 "Example of {value} is 'MIT'",
                                  "Expected either AND or OR, found a license or exception name: '{value}'.");
     DECLARE_AND_REGISTER_MESSAGE(
         LicenseExpressionUnknownLicense,
         (msg::value),
-        "",
+        "Example of {value} is 'unknownlicense'",
```

Comment:
```suggestion
        "{value} is the unrecognized license name supplied by the user, such as 'unknownlicense'.",
```

I think we should be clear that this comes from the user, not from the tool.